dev-requirements.txt

```diff
@@ -8,4 +8,5 @@ pre-commit==1.21.0
 cfgv==2.0.1
 detect-secrets==0.13.0
 
-pytest-cov~=2.7.1
\ No newline at end of file
+pytest-cov~=2.7.1
+alembic==1.4.3 # once we remove py35, we can use more recent versions (1.5.8)
```

Comment:
Is this a dev-requirement or requirement? It seems like it should be a requirement but on the chance that it isn't we need to update [the build script in datatools](https://github.com/NCI-GDC/tungsten/blob/feat/DEV-566-add-alembic-to-datatools/salt/srv/services/datatools/bin/build#L12) to include dev-requirements.txt.

Reply:
gdc-ng-models isn't pip-compilized yet, and there's a version conflict here because of which versions of python gdc-ng-models is supporting. the latest version 1.5.8 only suppots py27, py36+ (not py35), and the last version to support py35 is 1.4.3. So ideally the versions would sync up, but it's nice to have the latest in datatools since thats the one that will be used for migrations. I wasn't sure if alembic here is a dev requirement or not, its not used in code (yet), and its here so that users can create new migration scripts from the command line.

Reply:
Since this repo isn't pip-compiled yet we can start emulate it in the intermediate. We can add a range to setup.py and a specific version to requirements.txt. That way it can be included implicitly when we run a pip-compile update in `data-tools`.